.deployment.config.json

```diff
@@ -70,7 +78,7 @@
       "team_jenkins": {
         "disabled": true,
         "job_name": "ui-kit-qa-release",
-        "qa": {
+        "dev": {
           "disabled": false
         }
       }
```

Comment:
It's a little weird, but now the `JenkinsfileQARelease` file is executed in the `dev` branch. I made do this since `JenkinsfileQARelease` is responsible for creating a release bump commit. The above "manual" step replaces the old dev approval.